test/board_test.rb

```diff
@@ -22,6 +24,8 @@ def test_it_has_cells
     assert_instance_of Cell, @board.cells["AI"].class
```

Comment:
I'm blind, but I think you may've put an 'I' instead of a '1' for your cell location. :)

Reply:
Oh wait, that may've been something you already removed...